app/views/users/backup_code_setup/index.html.erb

```diff
@@ -1,5 +1,6 @@
 <%= render PageHeadingComponent.new.with_content(@presenter.title) %>
-<p class="mt-tiny margin-bottom-4"><%== @presenter.description %></p>
+
+<p class="margin-bottom-4"><%== @presenter.description %></p>
```

Comment:
(not related to this PR so feel free to skip) but the `<%==` surprised me

Looks like that comes from:
https://github.com/18F/identity-idp/blob/69e9db5d4a683a38456ce4455fa0dd4d07109a18/app/presenters/backup_code_create_presenter.rb#L8-L10
which comes from: 
https://github.com/18F/identity-idp/blob/69e9db5d4a683a38456ce4455fa0dd4d07109a18/config/locales/forms/en.yml#L6-L11

so if we renamed the key to add `_html` we could probably remove the `<%==`

```suggestion
<p class="margin-bottom-4"><%= @presenter.description %></p>
```

Reply:
Seems like a good opportunity to switch some `<b>` to `<strong>` as well!

Reply:
Digging into this revealed quite a few other clean-up opportunities with this view/presenter, so I'm gonna save it for a quick follow-on pull request.